torch/csrc/distributed/c10d/ProcessGroupNCCL.cpp

```diff
@@ -725,7 +731,7 @@ void ProcessGroupNCCL::WorkNCCL::synchronizeInternal(
 bool ProcessGroupNCCL::WorkNCCL::wait(std::chrono::milliseconds timeout) {
   RECORD_PARAM_COMMS(
       static_cast<int>(this->seq_), // seq
-      std::make_tuple("", ""), // PG name tuple
+      std::make_tuple(pgName_, pgDesc_), // PG name tuple
```

Comment:
just for my understanding, is the motivation of this change so that (pgName_, pgDesc) can be logged from this line?

also, do we need to make these copies? i thought work has a reference to parent PG, but maybe not?

Reply:
Yes, I am working on PyTorch execution trace, "wait" op missed the process group info, so it does not know which collective to wait for. 

I can not find a reference to parent PG. Let me know if you find it.

Reply:
cc @fduwjj @shuqiangzhang Lmk if you guys want to change this later, it might be generally useful to make work hold a ref to parent.  But ill stamp for now.